Formula/i/i2p.rb

```diff
@@ -1,13 +1,13 @@
 class I2p < Formula
   desc "Anonymous overlay network - a network within a network"
   homepage "https://geti2p.net"
-  url "https://files.i2p-projekt.de/2.5.1/i2psource_2.5.1.tar.bz2"
+  url "https://github.com/i2p/i2p.i2p/releases/download/i2p-2.5.1/i2psource_2.5.1.tar.bz2"
```

Comment:
We should probably avoid using `CI-syntax-only` when the PR changes the formula `url`.

Reply:
but it is using the same sha? (we did that before too)

Reply:
```
$ brew fetch -s i2p
==> Downloading https://github.com/i2p/i2p.i2p/releases/download/i2p-2.5.1/i2psource_2.5.1.tar.bz2
==> Downloading from https://objects.githubusercontent.com/github-production-release-asset-2e65be/4457770/27e9a75e-57b0-4dd6-af21-601b8c9a506c?X-Amz-Algorithm=AWS4-HMAC-S
################################################################################################################################################################### 100.0%
Downloaded to: /Users/rui/Library/Caches/Homebrew/downloads/f5b8c9c24a2a07081f0dd427ee08c2be5713504ee2beffd9c8d3e4462eb46d22--i2psource_2.5.1.tar.bz2
SHA256: 4bc7e59ee0036389a0f76fc76b2303eeae62bf6eaaf608c9939226febf9ddeae
```